### DISCUSSION TITLE
set default queue type as quorum in vhost

### DISCUSSION THREAD

**Discussion by arunkumar-sdworx** [rabbitmq/rabbitmq-server#10998] [Other] [answered]
### Is your feature request related to a problem? Please describe.

I am migrating from old rabbitmq version 3.8 to 3.13 and move all the queues to quorum queues. when i set manually 

`rabbitmqctl update_vhost_metadata / --default-queue-type quorum`

then this gets changed to classic when the container restarts.

i can do some customization to change to update it again when the container starts. 

### Describe the solution you'd like

i feel a definitions file supporting the queuetype is better 

     "vhosts": [
		{
			"name": "/",
                         "queuetype": "quorum"
		}
	]

**Comment by michaelklishin** [comment, marked as answer]
This is already the case since 3.11.x but the key name is not "queuetype". See https://github.com/rabbitmq/rabbitmq-server/issues/7734.
